Update LLM (#582)
kuzudb/kuzu-docs: src/content/docs/extensions/llm.mdx

| `google-gemini` | `GOOGLE_GEMINI_API_KEY` |
| `open-ai` | `OPENAI_API_KEY` |
| `voyage-ai` | `VOYAGE_API_KEY` |

> | `ollama` | Optional: `OLLAMA_URL` |

Reviewer: ```suggestion
| `ollama` | `OLLAMA_URL` |
```

:::note[Note]

> Ollama endpoint configuration works as follows. If an endpoint argument is provided it will be used to configure the endpoint. 
> If an endpoint is not provided, and OLLAMA_URL is set, the OLLAMA_URL variable will be used. Otherwise, the default endpoint, 
> `http://localhost:11434`, will be used.

Reviewer: ```suggestion
`OLLAMA_URL` is optional for `ollama`. You can also choose to configure the endpoint
as an argument to the `CREATE_EMBEDDING` function, as explained above.
```

RETURN CREATE_EMBEDDING("Hello world", "google-gemini", "gemini-embedding-exp-03-07");

// Required REGION
RETURN CREATE_EMBEDDING("Hello world", "google-vertex", "gemini-embedding-001", "us-central1");

> // Optional DIMENSIONS and Required REGION.

Reviewer: ```suggestion
// Optional DIMENSIONS and required REGION.
```
